sass/components/_sidebar.scss

```diff
@@ -0,0 +1,38 @@
+@mixin sidenavbar() {
```

Comment:
Nếu không có biến truyền vào, thì sử dụng %Placeholders

Reply:
qua em có sử dụng rồi mà nó kh cho anh ạ, nó báo là k cho. Nó báo như này ạ "you may not @extend selectors across media queries" nên em mới sử dụng mixin